openpype/tools/workfiles/window.py

```diff
@@ -94,6 +95,18 @@ def _on_save_click(self):
         self._on_note_change()
         self.save_clicked.emit()
 
+    def get_user_name(self, file):
+        """Get user name from file path"""
+        # Only run on Unix because pwd module is not available on Windows.
+        # NOTE: we tried adding "win32security" module but it was not working
+        # on all hosts so we decided to just support Linux until migration
+        # to Ayon
+        if platform.system() != "Windows":
+            import pwd
+
+            filestat = os.stat(file)
+            return pwd.getpwuid(filestat.st_uid).pw_name
```

Comment:
```suggestion
        if platform.system() != "linux":
            return None
        import pwd

        filestat = os.stat(file)
        return pwd.getpwuid(filestat.st_uid).pw_name
```

Reply:
`pwd` also works on Mac right?

Reply:
yeah it definitely works in Mac as well, just tested it on a Mac of ours:
```
vfx-studiomac-82:/ farrizabalaga$ python
Python 2.7.16 (v2.7.16:413a49145e, Mar  2 2019, 14:32:10) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pwd
>>> import os
>>> filestat = os.stat("/Scripts/mount_transfer_storage.sh")
>>> pwd.getpwuid(filestat.st_uid).pw_name
'root'
```

Reply:
Ok, changed the condition to match the usecase.